src/encoder.rs

```diff
@@ -1473,17 +1473,27 @@ pub fn write_tx_tree<T: Pixel>(
 
     for p in 1..3 {
       ts.qc.update(qidx, uv_tx_size, false, fi.sequence.bit_depth, fi.dc_delta_q[p], fi.ac_delta_q[p]);
-      let tx_bo = BlockOffset {
-        x: tile_bo.x  - ((bw * tx_size.width_mi() == 1) as usize),
-        y: tile_bo.y  - ((bh * tx_size.height_mi() == 1) as usize)
-      };
 
-      let po = tile_bo.plane_offset(&ts.input.planes[p].cfg);
-      let (_, dist) =
-        encode_tx_block(fi, ts, cw, w, p, tx_bo, luma_mode, uv_tx_size, uv_tx_type,
+      for by in 0..bh_uv {
```

Comment:
Ohh is this for 32x32 chroma tx limitation on 64x64 txs?

Reply:
I guess so. I naively copied it from `write_tx_blocks()` in case that was the missing detail and it seems it was.